src/main/java/ratpack/zipkin/ServerTracingModule.java

```diff
@@ -115,6 +120,28 @@ else if (config.spanCollector != null) {
     return braveBuilder.build();
   }
 
+  private static byte[] getHostAddress() {
+    try {
+      return hostAddress().orElse(InetAddress.getLoopbackAddress()).getAddress();
+    } catch (SocketException e) {
+      return InetAddress.getLoopbackAddress().getAddress();
```

Comment:
this looks a little goofy to have the `InetAddress.getLoopbackAddress()` repeated in both sides but it looks like it should work fine.